components/pagination/stories/pagination.stories.js

```diff
@@ -58,7 +79,11 @@ export default {
 	},
 };
 
+/**
+ * The default listing/page variant uses buttons for each page number. 
+ */
 export const Default = PaginationGroup.bind({});
+Default.storyName = "Default (Listing)";
```

Comment:
```suggestion
Default.storyName = "Default (listing)";
```

Is listing a name or a descriptor? I think it should probably be lowercase since it describes the variant.